apis/server/container_bridge.go

```diff
@@ -99,41 +102,43 @@ func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req
 		All: httputils.BoolValue(req, "all"),
 	}
 
-	metas, err := s.ContainerMgr.List(ctx, func(meta *mgr.ContainerMeta) bool {
+	cons, err := s.ContainerMgr.List(ctx, func(c *mgr.Container) bool {
 		return true
 	}, option)
 	if err != nil {
 		return err
 	}
 
-	containerList := make([]types.Container, 0, len(metas))
+	containerList := make([]types.Container, 0, len(cons))
 
-	for _, m := range metas {
-		status, err := m.FormatStatus()
+	for _, c := range cons {
+		status, err := c.FormatStatus()
 		if err != nil {
 			return err
 		}
 
-		t, err := time.Parse(utils.TimeLayout, m.Created)
+		t, err := time.Parse(utils.TimeLayout, c.Created)
 		if err != nil {
 			return err
 		}
 
-		container := types.Container{
-			ID:         m.ID,
-			Names:      []string{m.Name},
-			Image:      m.Config.Image,
-			Command:    strings.Join(m.Config.Cmd, " "),
-			Status:     status,
-			Created:    t.UnixNano(),
-			Labels:     m.Config.Labels,
-			HostConfig: m.HostConfig,
+		var netSettings *types.ContainerNetworkSettings
+		if c.NetworkSettings != nil {
+			netSettings = &types.ContainerNetworkSettings{
+				Networks: c.NetworkSettings.Networks,
+			}
 		}
 
-		if m.NetworkSettings != nil {
-			container.NetworkSettings = &types.ContainerNetworkSettings{
-				Networks: m.NetworkSettings.Networks,
-			}
+		container := types.Container{
```

Comment:
`types.Container` and `ContainerMgr.Container` may be confused, can we named different to distinct this two containers  struct ?

Reply:
Updated, PTAL @HusterWan